src/index.md

```diff
@@ -985,35 +985,109 @@ For the purposes of brevity this document will use the term "bug" to refer to bo
-
-This example was written in the JDK 17 development time frame which is why the boot JDK used here is JDK 16. To build JDK N, use JDK N-1 as the boot JDK. Note that the download link used here points to a Linux x64 JDK 16.
+The examples below show the steps taken to build the JDK source code. Please see [Cloning the JDK](#cloning-the-jdk) for information on how to download it. These examples were written in the JDK 17 development time frame which is why the boot JDK used here is JDK 16. Note that the download links used here point to JDK 16 bundles. To build JDK N, use JDK N-1 as the boot JDK.
 
-This is just an example log, don't copy the `apt-get install` line. Instead run `sh ./configure` once you have the boot JDK installed to see what packages you actually need on your system.
+The configure script will tell you what additional packages you need. In this first example several packages were needed since this build was performed on a clean Ubuntu installation. The configure script was run several times to get all the dependencies, but only the commands actually needed to get the JDK built are included in the log. This is just an example log, don't copy the `apt-get install` line. Instead run `sh ./configure` to see what packages you actually need on your system.
 
-    $ git clone https://github.com/openjdk/jdk.git
     $ wget https://download.java.net/java/GA/jdk16/7863447f0ab643c585b9bdebf67c69db/36/GPL/openjdk-16_linux-x64_bin.tar.gz
     $ tar xzf openjdk-16_linux-x64_bin.tar.gz
     $ sudo apt-get install autoconf zip make gcc g++ libx11-dev libxext-dev libxrender-dev libxrandr-dev libxtst-dev libxt-dev libcups2-dev libfontconfig1-dev libasound2-dev
     $ cd jdk
     $ sh ./configure --with-boot-jdk=$HOME/jdk-16/
     $ make images
 
-In this case the built JDK can be found in `build/linux-x86_64-server-release/jdk`. The exact path depends on your build platform and selected configuration.
+The built JDK can be found in `build/linux-x86_64-server-release/jdk`. The exact path depends on your build platform and selected configuration.
+
+The second example is from a clean (newly installed) Mac running MacOS Big Sur. Please note that in this case there are some steps taken outside of the terminal. Here [Mac Ports](https://www.macports.org) is used to install `autoconf`. `autoconf` can also be installed using [Homebrew](https://brew.sh) and surely through other sources as well.
+
+    $ curl https://download.java.net/java/GA/jdk16.0.1/7147401fd7354114ac51ef3e1328291f/9/GPL/openjdk-16.0.1_osx-x64_bin.tar.gz --output openjdk-16.0.1_osx-x64_bin.tar.gz
+    $ tar xzf openjdk-16.0.1_osx-x64_bin.tar.gz
+    $ # Get XCode from Appstore
```

Comment:
I think this should be a separate text paragraph, not a comment in what looks like a cookbook script to building the JDK. Getting XCode is kind of a problem. You can get it from the App Store; but sometimes the App Store version is too new for the OS version you have, so you have to get an older XCode. But of course if XCode is too old then it can't build the JDK. Obviously we can't write a whole section that covers all these issues. Plus, it changes from time to time as Apple releases new versions of things. I'd suggest having a separate paragraph that talks about getting XCode from the App Store, or the possible need to get older versions from Apple's developer site. I would avoid putting too many details in this document. Probably the best fallback is to tell people to ask on the mailing list if they have trouble with this step.

Reply:
I separated out the XCode info and put it in the text above, as a precondition to building.